test/md055-missing-pipes.md

```diff
@@ -0,0 +1,19 @@
+# MD055 Missing Borders
```

Comment:
This is great. I'd like to see some multi-row tables, too, with missing pipes on the first/middle/last rows. Let's be sure the line numbers are correct.